configure_metrics_agent.sh

```diff
@@ -189,6 +191,11 @@ function set_attributes() {
         sudo find "${directory}" -type d -exec chown mon-agent:mon-agent {} +
     done
 
+    for config_file in "${MON_CONF_FILES[@]}"
+    do
+        chmod 0664 "${config_file}"
```

Comment:
You're granting too many permissions here. `agent.yaml` shouldn't be readable for everyone.

I think it would be better to set `0644` for [masca-agent.service](https://github.com/monasca/monasca-agents-installer/pull/42/commits/f684b56a7c9b56c23b03c3db48e8008bd5b41de7#diff-2afa8208c7cef4f6c6a37123bcc5aca4R176) and `0640` for [supervisor.conf](https://github.com/monasca/monasca-agents-installer/pull/42/commits/f684b56a7c9b56c23b03c3db48e8008bd5b41de7#diff-2afa8208c7cef4f6c6a37123bcc5aca4R144).

Reply:
Done